src/plugins/items/src/components/visualizer.rs

```diff
@@ -179,6 +214,41 @@ mod tests {
 
 		let lookup = Visualizer::<_View, _Key>::new([(Name::from("B"), Entity::from_raw(100))]);
 
-		assert_eq!(Some(&Entity::from_raw(100)), lookup.get(&_Key));
+		assert_eq!(Some(&Entity::from_raw(100)), lookup.get(&_Key::A));
+	}
```

Comment:
I think this is not needed. As far as I can see, it is impossible to write a failing test here, if it succeeds with one key already, because the key is generic. The `get` method has no way of using a different key than the provided one and has no meaningful insights into the state/contents of the key.